### PR TITLE
Added Sendinblue as a provider

### PR DESCRIPTION
**Pull request by galezra**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Added SendinBlue as a provider.

- **What is the current behavior?** (You can also link to an open issue here)
#79 

- **What is the new behavior (if this is a feature change)?**
Use SendinBlue as a email provider

- **Other information**:
Used the [SendinBlue typescript repo ](https://github.com/sendinblue/APIv3-typescript-library#readme)to create the provider

### CODE REVIEW

**Comment by scopsy** [comment]
From a quick pick look amazing @galezra ! I'll take a closer look in the near couple of days. Do you want maybe to already add the attachments support? Just merge from v0.3.0 and move your pr to point to it instead of master.

**Comment by galezra** [comment]
Sure, I'll add the attachment support and resolve the conflict

**Comment by scopsy** [comment]
@galezra This looks great just waiting on the attachments and we can merge :)

**Comment by galezra** [comment]
> @galezra This looks great just waiting on the attachments and we can merge :) Let me know if you need any help with it

Done

File: providers/sendinblue/src/lib/sendinblue.provider.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  ): Promise<ISendMessageSuccessResponse> {
+    const email = new SendSmtpEmail();
+    email.sender = { email: options.from || options.from };
+    email.to = getFormattedTo(options.to);
+    email.subject = options.subject;
+    email.htmlContent = options.html;
+    email.textContent = options.text;
+    email.attachment = options.attachments?.map((attachment) => ({
+      name: attachment?.name,
+      content: attachment?.file?.toString('utf8'),
```
Comment (scopsy) [inline providers/sendinblue/src/lib/sendinblue.provider.ts:42]: @galezra Have you tried doing a test case of sending it here? I'm not sure about the utf8 string here, in the docs looks like they say it should be either a relative path or a base64 encoding. ( https://github.com/sendinblue/APIv3-nodejs-library/blob/master/docs/SendSmtpEmail.md )

If you haven't tried it our I can maybe create a free account there later to try and see if the attachments are sent and can be opened on the other side.

**Review by scopsy — COMMENTED** [review]
This looks fantastic and I think we are ready to merge. Just a small questions about the `utf8` there

**Comment by galezra** [comment]
@scopsy You're right, utf8 encoding wasn't necessary. I fixed it and pushed the changes

**Pull request merged**